homeassistant/components/mqtt/__init__.py

```diff
@@ -84,14 +85,20 @@ def valid_publish_topic(value):
 CLIENT_KEY_AUTH_MSG = 'client_key and client_cert must both be present in ' \
                       'the mqtt broker config'
 
+MQTT_PUBLISH_SCHEMA = vol.Schema({
```

Comment:
Please submit a fix for this @bestlibre or we'll have to rollback this PR before the next release (Friday).